apps/omg_eth/lib/omg_eth/release_tasks/set_contract.ex

```diff
@@ -60,8 +60,7 @@ defmodule OMG.Eth.ReleaseTasks.SetContract do
           plasma_framework_tx_hash: txhash_contract
         } = Jason.decode!(body, keys: :atoms!)
 
-        exit_period_seconds =
-          validate_integer(get_env("EXIT_PERIOD_SECONDS"), Application.get_env(@app, :exit_period_seconds))
+        min_exit_period = validate_integer(get_env("MIN_EXIT_PERIOD"), Application.get_env(@app, :min_exit_period))
```

Comment:
From the contract side, only `development` is allowed to be overridden by env variable. As a result, if once the config here and the one on `production` in contract get out synced, this way of getting `min_exit_period` might cause issue as the real effective one would always be in contract.

Might consider to fetch the real value from contract to override the `update_configuration` part instead.

see: https://github.com/omisego/plasma-contracts/blob/master/plasma_framework/migrations/config.js#L31